scripts/wait4.js

```diff
@@ -106,21 +106,14 @@ async function wrapWithRetries (method, waitJob, retries, waitTimeMs) {
 }
 
 /**
- * @function getRC
- * @description - create RC config instance
+ * @function getConvictConfig
+ * @description - create convict config instance
  * @param {object} waitJob
  */
-function getRC (waitJob) {
+function getConvictConfig (waitJob) {
   // acquire rc parameters
-  const namespace = (waitJob.rc && waitJob.rc.namespace) || 'CLEDG'
-  const configPath = (waitJob.rc && waitJob.rc.configPath) || '../config/default.json'
-
-  // require rc to deliver config
-  try {
-    return require('rc')(namespace, require(configPath))
-  } catch (err) {
-    return waitJob.rc || {}
-  }
+  const configPath = '../config/integration_db.json'
```

Comment:
maybe we should resolve config path using NODE_ENV variable, so we will be able to run integration tests with different configurations for knex...

Reply:
Makes sense but I will have to use `process.env.NODE_ENV` because I cannot import typescript files here - so I cannot retrieve it from Convict unless we make another javascript version